libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -125,13 +147,18 @@ void AC_Avoid::adjust_speed(float kP, float accel, float heading, float &speed,
     Vector2f vel_xy;
     vel_xy.x = cosf(heading) * speed * 100.0f;
     vel_xy.y = sinf(heading) * speed * 100.0f;
-    adjust_velocity(kP, accel * 100.0f, vel_xy, dt);
+
+    bool backing_up  = false;
+    adjust_velocity(kP, accel * 100.0f, vel_xy, backing_up, dt);
+
+    float is_reverse;
```

Comment:
`const float is_reverse = backing_up ? -1.0f : 1.0f;` maybe ?

Reply:
we should probably rename, "is_reverse" to something else. like reverse_factor, or something..